screens/Home.screen.js

```diff
@@ -0,0 +1,20 @@
+import AppScreen from './app.screen';
+const SELECTORS = {
+  HOME_SCREEN: driver.isAndroid
+    ? '-android uiautomator: new UiSelector().description("Home-screen")'
+    : '~Home-screen',
+  TEXT: driver.isAndroid
+    ? '-android uiautomator: new UiSelector().text("{TEXT}")'
+    : '//XCUIElementTypeStaticText[@name="{TEXT}"]'
+};
```

Comment:
This code could be a little difficult to understand to someone new. Maybe put a comment with a short description about what it does?

Reply:
done